graylog2-web-interface/src/components/streamrules/HumanReadableStreamRule.jsx

```diff
@@ -23,7 +20,21 @@ class HumanReadableStreamRule extends React.Component {
     return undefined;
   };
 
+  _inputName = (inputId) => {
+    const { inputs } = this.props;
+
+    return inputs.find((input) => input.id === inputId);
+  }
+
   _formatRuleValue = (streamRule) => {
+    if (String(streamRule.type) === String(this.MATCH_INPUT)) {
+      const input = this._inputName(streamRule.value);
+
+      if (input) {
+        return `${input.title} (${input.name}: ${input.id})`;
+      }
```

Comment:
Right now, if the input does not exist/is not started, only the id is shown. IMO we should also show an indicator, something like `<deleted input>`, like we do in message details.

Reply:
Best idea ever! Now you can remove the rules with deleted input!